src/capture.js

```diff
@@ -45,6 +45,9 @@ if (n.msPointerEnabled) {
     dispatcher.releaseCapture(pointerId, this);
   };
 }
+h = function hasPointerCapture(pointerId) {
+  return !!dispatcher.captureInfo[pointerId];
+};
```

Comment:
Ah, of course! Thank you for catching that. Not quite as straight forward as I had hoped.